sdk/core/azure-core/azure/core/pipeline/policies/_utils.py

```diff
@@ -80,6 +79,7 @@ def get_retry_after(response):
     :return: Value of Retry-After in seconds.
     :rtype: float or None
     """
+    from requests.structures import CaseInsensitiveDict
```

Comment:
We probably want to use https://github.com/Azure/azure-sdk-for-python/blob/9711f486c1a6e446e96f5d6e43df10da9d05ebff/sdk/core/azure-core/azure/core/pipeline/transport/_base.py#L89 instead